src/components/Button/Button.tsx

```diff
@@ -1,12 +1,79 @@
 interface ButtonProps {
   text: string;
   type: "anchor" | "button";
+  className?: string;
 }
+import { Link } from "react-router-dom";
+const register = "/register";
 
-export function Button({ text, type }: ButtonProps) {
+export function ButtonLogin({ text, type }: ButtonProps) {
+  const login = "/login";
+
+  if (type === "anchor") {
+    return (
+      <Link
+        to={login}
+        className="flex h-10 w-24 items-center justify-center rounded-lg bg-white font-bold text-blue-600"
```

Comment:
1. These classes are the same as in the `ButtonRegister` component, so, they're the same component (Eg. `Button` as a generic button component).
2. Don't use a hardcoded width and height for buttons, just add padding: 

![image](https://github.com/hawks-atlanta/frontend-react/assets/62714297/e11b3783-ca80-4dad-9a61-4ebc4bd4c9c3)